src/assets/scss/introjs.scss

```diff
@@ -71,13 +71,6 @@
 .introjs-prevbutton {
   display: none;
 }
-a:not([href]) {
-  color: $white;
-
-  &:hover {
-    color: $white;
-  }
-}
```

Comment:
Removing this styling creates a design discrepancy for the button in the walkthrough. The text color is blue and not white.
![Screen Shot 2021-03-19 at 3 03 46 PM](https://user-images.githubusercontent.com/54298311/111830437-4fe04f00-88c4-11eb-84fb-be2b7b71b3a7.png)

Designs:
![Screen Shot 2021-03-19 at 3 05 09 PM](https://user-images.githubusercontent.com/54298311/111830538-828a4780-88c4-11eb-87d2-ee82eefdf5dc.png)

Reply:
😢 will fix soon

Reply:
i wasnt able to verify if i fixed this because the walkthrough isn't triggered after you go through onboarding. I can't tell if this is a bug or a feature.

Reply:
This is a bug fix - now onboarding is only triggered the very first time a user completes the walkthrough as was intended.

If you want to trigger the walkthrough, you can either delete your firebase data on dev (let me know if you need any more instructions on how to do that) or you can locally change the if statement in the `endOnboarding()` method in `Dashboard.vue` to `if (true)` instead of `if (!this.isMobile && !this.isEditingProfile)` so that it will trigger everytime you go through onboarding as before.